packages/contracts/contracts/L2/ValidatorRewardVault.sol

```diff
@@ -122,6 +123,24 @@ contract ValidatorRewardVault is FeeVault, Semver {
         );
     }
 
+    /**
+     * @notice Withdraws all of the sender's balance to L2.
+     */
+    function withdrawToL2() external {
+        uint256 balance = rewards[msg.sender];
+        rewards[msg.sender] = 0;
+
+        unchecked {
+            totalReserved -= balance;
+            totalProcessed += balance;
+        }
+
+        emit Withdrawal(balance, msg.sender, msg.sender);
```

Comment:
Since this event is from `FeeVault` and it originally means the withdrawal event to L1, how about add new event in this contract that means withdrawal event to L2?

Reply:
The `Withdrawal` event means that a withdrawal request has been made, rather than a withdrawal to L1. 
The event of withdrawing to L1 can be checked as an event emitted by L2 Bridge contract.